http/build.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+    compile 'io.undertow:undertow-core:2.0.13.Final'
```

Comment:
nitpick: there is a newer version. I tried with 2.1.1.Final and the tests are passing.